server/config/config_test.go

```diff
@@ -1,9 +1,53 @@
 package config
 
 import (
+	"gotest.tools/v3/assert"
```

Comment:
i perfer to remove this dep (i want to keep the deps clean), since we dont' have to much tests in golang side, please use `t.fatal` instead.

Reply:
Sure!

Reply:
thanks